includes/dart-integrations/firebase-remote-config.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Firebase Remote Config
+description: "Learn more about the Sentry Firebase Remote Config integration for the Dart SDK."
+sidebar_order: 6
+platforms:
+  - flutter
+---
+
+The `sentry_firebase_remote_config` integration provides [Firebase Remote Config](https://firebase.google.com/docs/remote-config/get-started?platform=flutter) support for Sentry thus providing insight into feature flag evaluations.
+
+## Behavior
+
+- Adding the `SentryFirebaseRemoteConfig` integration will automatically track feature flag evaluations.
+- Only boolean values are supported.
+
+## Prerequisites
+
+1. [Sentry SDK](/platforms/dart/#configure) version `9.0.0` or higher is installed.
+2. Firebase Remote Config is set up.
+
+## Install
+
+To use the `SentryFirebaseRemoteConfig` integration, add the `sentry_firebase_remote_config` dependency.
+
+```yml {filename:pubspec.yaml}
+dependencies:
+  sentry: ^9.0.0
+  sentry_firebase_remote_config: ^9.0.0
+```
+
+## Configure
+
+Add the `SentryFirebaseRemoteConfig` integration to the Sentry SDK initialization. The integration will automatically call `await remoteConfig.activate();` when the config is updated.
+If you don't want this behavior, you can set `activateOnConfigUpdated` to `false`.
+
+```dart
+await SentryFlutter.init(
+  (options) {
+    options.addIntegration(
+      SentryFirebaseRemoteConfigIntegration(
+        firebaseRemoteConfig: firebaseRemoteConfig,
+        activateOnConfigUpdated: false,
+      ),
+    );
+  },
+);
+```
```

Comment:
> If you don't want this behavior, you can set `activateOnConfigUpdated` to `false`.

Can we split this up into another section? 

The first is `Configure` and then the subsection would be something like `Remote Config Update` and then explain the `activateOnConfigUpdated` there.

It's confusing to me that `activateOnConfigUpdated: false` is being set here